2023-11-03 09:56:47,571 [main.py         | 48] INFO: Start to run instance_10
2023-11-03 09:56:47,572 [simulate_api.py | 60] INFO: Start time of the simulator: 2023-11-03 00:00:00, time interval:  600.00
2023-11-03 09:56:47,581 [input_utils.py  | 47] INFO: Get 154 factories
2023-11-03 09:56:48,536 [input_utils.py  | 51] INFO: Get 23562 routes
2023-11-03 09:56:48,571 [input_utils.py  | 56] INFO: Get 5 vehicles
2023-11-03 09:56:48,583 [input_utils.py  | 60] INFO: Get 100 orders
2023-11-03 09:56:48,583 [simulate_api.py | 87] INFO: Initial position of V_1 is e2d5093fbe36431f8986ddb0e1c586be
2023-11-03 09:56:48,584 [simulate_api.py | 87] INFO: Initial position of V_2 is 7fe14b93f0f04ee7a994ef5b2c1fdb72
2023-11-03 09:56:48,584 [simulate_api.py | 87] INFO: Initial position of V_3 is fa366fc87a124d32926daa5bb093129f
2023-11-03 09:56:48,585 [simulate_api.py | 87] INFO: Initial position of V_4 is e47399648fa842b2b8f80094343d8091
2023-11-03 09:56:48,585 [simulate_api.py | 87] INFO: Initial position of V_5 is becb4f85393540b287e7329758b8d832
2023-11-03 09:56:48,586 [simulate_environment.py | 99] INFO: **************************************************
2023-11-03 09:56:48,586 [simulate_environment.py | 103] INFO: cur time: 2023-11-03 00:10:00, pre time: 2023-11-03 00:00:00
2023-11-03 09:56:48,586 [simulate_environment.py | 140] INFO: Start to update the input of 2023-11-03 00:10:00
2023-11-03 09:56:48,588 [simulate_environment.py | 167] INFO: Get 0 unallocated order items, 0 ongoing order items, 0 completed order items
2023-11-03 09:56:50,183 [simulate_environment.py | 253] INFO: 2023-11-03 00:10:00, Item 0010370001-1: state = 0 < 2, we can not finish the simulation
2023-11-03 09:56:50,184 [main.py         | 55] ERROR: Failed to run simulator
2023-11-03 09:56:50,185 [main.py         | 56] ERROR: Error: 'NoneType' object does not support item assignment, Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\main.py", line 51, in <module>
    score = simulate(Configs.factory_info_file, Configs.route_info_file, instance)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\src\simulator\simulate_api.py", line 94, in simulate
    simulate_env.run()
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\src\simulator\simulate_environment.py", line 132, in run
    self.total_score = Evaluator.calculate_total_score(self.history, self.route_map, len(self.id_to_vehicle))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\src\utils\evaluator.py", line 33, in calculate_total_score
    total_distance = Evaluator.calculate_total_distance(history.get_vehicle_position_history(), route_map)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\src\utils\evaluator.py", line 54, in calculate_total_distance
    destination_dict[vehicle_id] = distance
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^
TypeError: 'NoneType' object does not support item assignment

